list.txt

```diff
@@ -18,3 +18,7 @@
 *://issueexplorer.com/*
 *://uonfu.com/*
 *://gitcodeask.cc/*
+*://grabthiscode.com/*
+*://*.hotexamples.com/*
+*://hotexamples.com/*
```

Comment:
afaict this is redundant when you have a wildcard subdomain rule

Reply:
I think its just regex right? like `*://*.hotexamples.com/*` != `*://hotexamples.com/*` in regex at least

Reply:
forget that https://iorate.github.io/ublacklist/docs/advanced-features

Reply:
you just got pranked :^)